### ISSUE TITLE
[1.7.6.4] Can't open Advanced->Information and perform update

### ISSUE THREAD

**Issue by banan44** [PrestaShop/PrestaShop#19736]
Hello,
Some time ago I have noticed in my Dashboard a notification that there is an update to 1.7.6.5 available. I went to 1-click upgrade module like I always do when there is a new PS version but this time after a few minutes I got an error:

503 Service Unavailable
The server is temporarily unable to service your request due to maintenance downtime or capacity problems. Please try again later.

I did not start the upgrade yet to see that error. It appeared when I was entering the configuration page of 1-click upgrade. That was strange because I update my shop regularly since 1.7.2. So far I have never had such error. 1-click upgrade always opened with no problem and I was able to perform upgrade. Unfortunately I did not have time to investigate that more that day. 

Today I wanted to get a second try but first I have manually disabled almost all statistics modules (like Shop search, Data mining, etc.) in case these might caused the previous problem. Then I went to Advanced Parameters -> Information to get all information in case 503 problem occurs again. But it turned out that I cannot access the Information page as well. I have the same error 503 Service Unavailable on Information page as well. That's disturbing. When I upgraded from 1.7.6.3 to 1.7.6.4 I did not have these issues. After visiting the Information page and seeing the 503 error I went to my server to check the logs. Can you check them and try to help me resolve that? I have censored some parts for security reasons. In case they are needed I can send the original log in a private message or email. I have also visisted almost all pages and settings in Back Office to check if the 503 error occurs on other pages but it did not. It occurs only on Information page and 1-click upgrade module.

[logs.txt](https://github.com/PrestaShop/PrestaShop/files/4775632/logs.txt)

**Comment by rdy4ever** [comment]
At a firs glance I'd say you have a problem with file permissions on the server. You need to connect via ftp and change file permissions to 0755 for folders and 0644 for files, **_recursively_**. I'd suggest to use [Filezilla](https://filezilla-project.org/) to do that. [Here's a tutorial](https://help.vodien.com/articles/change-file-permissions-using-filezilla/). Remember to check "Recurse into subdirectories". 
Also, you could try to manually remove the contents of `var/cache` folder (`prod` and `dev` folders) in cPanel file manager. But this won't fix your error for the long run, so also do the file permission change.

**Comment by matthieu-rolland** [comment]
Could you please provide us with more info? We need more details to understand how we can reproduce your issue:

- host
- server setup and configuration
- PrestaShop version (source)
- debug mode report
- PHP error logs
- apache error log
- Javascript console log
- screenshots
- ...

You don't know how to get this information? Please read the following article: [How to create a bug report](http://build.prestashop.com/howtos/misc/how-to-create-bug-report/).

Thank you!

**Comment by banan44** [comment]
@rdy4ever, @matthieu-rolland 

Thanks for quick reply to my problem. I have set all folders to 0755 and all files to 0644 using Filezilla. I have cleared var/cache folder. Unfortunately, I still cannot access BO -> Advanced Parameters -> Information page. Still after a while it shows me 503 Service Unavailable. 

I have also other Prestashop installment on the same server which I use for testing purposes. I can open Information page with no problem on that PS installment. But this is Prestashop 1.7.6.0 version. I can update it to 1.7.6.4 for check if the problem also occurs here.


Server information Linux #1 SMP Wed Oct 2 07:53:12 EDT 2019 x86_64
MySQL version: 10.1.45-MariaDB-cll-lve 
Server software version: Apache 
PrestaShop version - 1.7.6.4 as indicated in the title 
PHP version: 7.2
Error log - original logs attached in the first post. Below are logs from today's attempt to open Information page

[logs 16.06.2020.txt](https://github.com/PrestaShop/PrestaShop/files/4789137/logs.16.06.2020.txt)

**Comment by florine2623** [comment]
Hi @banan44 ,
503 error are server error.
In your log file you have a timeout error. You should contact your hosting provider directly. 
Thanks!

**Comment by banan44** [comment]
Hi @florine2623 
I think it is not normal that opening Information page takes over 300 seconds. I have increased that number to 500 (server maximum) and still I cannot access Information page. While on my second Prestashop installment (my Prestashop for test purposes) on the same server, opening Information page takes about 5 seconds at max. Same server, just different folder, different domain and different PS version - 1.7.6.0. My original shop with the problem is PS 1.7.6.4 and I did not have that issue on 1.7.6.3

I cannot perform update because when I enter 1-Click Upgrade module I also have 503 error which in logs shows connection timeout. I think there is something wrong with the files because I have noticed that both pages (Information page and 1-Click Upgrade) checks if all files on the server are OK. 

EDIT
I do not know what happened but a few minutes ago I was able to open 1-Click Upgrade and I have performed an upgrade to 1.7.6.5 version. Now everything works correctly.
